rubygem-sprockets-rails/rubygem-sprockets-rails.spec

```diff
@@ -5,17 +5,17 @@
 %global gem_name sprockets-rails
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 2.2.4
-Release: 2%{?dist}
+Version: 2.3.3
+Release: 1%{?dist}
 Summary: Sprockets Rails integration
 Group: Development/Languages
 License: MIT
 URL: https://github.com/rails/sprockets-rails
-Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
```

Comment:
The Jenins slave that tito's usually run on is EL6 and it doesn't like those certificates, among other problems with HTTPS URLs.  It needs rebuilding.